catalog/transformer/catalog-transformer-streaming-lib/src/main/java/org/codice/ddf/transformer/xml/streaming/lib/SaxEventToXmlElementConverter.java

```diff
@@ -78,15 +86,18 @@ public SaxEventToXmlElementConverter() throws UnsupportedEncodingException, XMLS
      */
     public SaxEventToXmlElementConverter toElement(String uri, String localName, Attributes atts)
             throws XMLStreamException {
-
+        Map<String, String> scopedNamespaces = new HashMap<>();
+        scopeOfNamespacesAdded.push(scopedNamespaces);
         /*
          * Use the uri to look up the namespace prefix and append it and the localName to the start tag
```

Comment:
Since we're touching this code, should we extract private methods and give them meaningful names instead of having block comments (here and below)?